shading/coral-trino-parser/build.gradle

```diff
@@ -0,0 +1,22 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
```

Comment:
I'm not sure if it's a good idea to make the shaded libraries a separate module, and let the previous module depend on it (rather than just shade within the previous module directly)

But if we go this route, we need to also publish this shaded module (ref: https://imperceptiblethoughts.com/shadow/publishing/#publishing-with-maven-publish-plugin) , so that the downstream can continue to consume coral as a library.